### PR TITLE
Add Option for Scanning Alt Text

### LINKED ISSUE #1149: Avoid scanning on images and links for android
At least for mobile users of yomitan I think it can be a bit frustrating when every time pressing on images from news articles or from google images or links from streaming videos like youtube etc we get a popup. According to this user from the discord server it is because of the alt attribute:

image

### PR DESCRIPTION
**Pull request by MarvNC**
Resolves #1149

image

### CODE REVIEW

**Review by cashewnuttynuts — APPROVED** [review]

**Review by StefanVukovic99 — COMMENTED** [review]
Should probably update `options-schema.json` and add an update function in OptionsUtil.

**Comment by MarvNC** [comment]
> Should probably update `options-schema.json` and add an update function in OptionsUtil.

Done, not sure how to test though. It just worked for me without adding the function so is it necessary?

**Comment by Kuuuube** [comment]
You also need to bump the version in `test/options-util.test.js` and dw about doing more.

File: ext/js/data/options-util.js [hunk 1]
```diff
@@ -1348,6 +1350,17 @@ export class OptionsUtil {
         await this._applyAnkiFieldTemplatesPatch(options, '/data/templates/anki-field-templates-upgrade-v41.handlebars');
     }
 
+    /**
+     * - Added scanning.scanAltText
+     * @type {import('options-util').UpdateFunction}
+     * @param {import('settings').Options} options
```
Comment (StefanVukovic99) [inline ext/js/data/options-util.js:1356]: this param annotation shouldn't be there i think

**Review by StefanVukovic99 — APPROVED** [review]

File: ext/js/data/options-util.js (hunk 1 above)
Comment (MarvNC) [reply ext/js/data/options-util.js:1356]: Why not? If anything I was confused why it was `any` in all the other upgrade functions, could it break things by being typed due to different versions of the settings maybe?

**Review by StefanVukovic99 — APPROVED** [review]

**Pull request merged**